src/shared/Atlassian.Bitbucket.UI/ViewModels/CredentialsViewModel.cs

```diff
@@ -0,0 +1,94 @@
+using System.ComponentModel;
+using System.Windows.Input;
+using Microsoft.Git.CredentialManager;
+using Microsoft.Git.CredentialManager.UI;
+using Microsoft.Git.CredentialManager.UI.ViewModels;
+
+namespace Atlassian.Bitbucket.UI.ViewModels
+{
+    public class CredentialsViewModel : WindowViewModel
+    {
+        private readonly IEnvironment _environment;
+
+        private string _userName;
+        private string _password;
+
+        public CredentialsViewModel()
+        {
+            // For designer
```

Comment:
This empty public constructor must be present for the UI designer to be able to instantiate the view model so it can render things correctly.

See in the corresponding `CredentialsView.axaml`:

```xaml
<Design.DataContext>
    <vm:CredentialsViewModel/> <!-- This is invoking the empty ctor of the VM -->
</Design.DataContext>
```